app/views/dresses/new.html.erb

```diff
@@ -0,0 +1,4 @@
+<div class="container">
+  <h1 class="mt-3">Add a new dress</h1>
+  <%= render 'form' %>
```

Comment:
Just a question - does it need to be like
<%= render "form", dress: @dress %> 
or is just 'form' OK?

Reply:
Yes! I knew I forgot something at the end. Thx!